addons/web/static/src/core/template_inheritance.js

```diff
@@ -178,6 +185,7 @@ function modifyAttributes(target, operation) {
 
         if (value) {
             target.setAttribute(attributeName, value);
+            target.setAttribute("t-translation-context-" + attributeName, translationContext);
```

Comment:
target.setAttribute(`t-translation-context-${attributeName}`, translationContext);

Reply:
should we have a check of some sort when add or remove is set?